components/coding/studentPortal/Sidebar.tsx

```diff
@@ -81,6 +81,7 @@ export const Sidebar: React.FC = () => {
           {user && (
             <div className="w-full">
               <p className="w-full ml-4 font-bold">{user.displayName}</p>
+              <p className="ml-4 font-medium">Student</p>
```

Comment:
what if we just display the role from the redux slice here so it shows either "student" or "coach"? maybe just use a string method to make it upper case

Reply:
great!
I had set the redux to isRoleCoach and just realized that you expected it to be a string, so I changed the redux to save the actual string "student" or "coach" and then did the .toUpperCase()